js/client-chat.js

```diff
@@ -808,7 +808,25 @@
 					}
 				}), 'text');
 				return false;
-
+				
+			case 'regdate':
+				if (!target) target = app.user.get('userid');
+				target = toId(target);
+				var self = this;
+				$.get("http://pokemonshowdown.com/users/" + target + ".json",
```

Comment:
The two things I mentioned above will need to be changed before I can merge this.